src/Tizen.NUI/src/public/Events/PanGestureDetector.cs

```diff
@@ -367,6 +379,18 @@ public uint GetMaximumTouchesRequired()
             return ret;
         }
 
+        /// <summary>
+        /// Retrieves the maximum age for the pan gesture motion as milliseconds.
```

Comment:
in miliseconds ?

Reply:
https://en.wikipedia.org/wiki/Millisecond
Milli (two l) is right word I think.